client/driver/qemu.go

```diff
@@ -436,11 +445,9 @@ func (h *qemuHandle) Signal(s os.Signal) error {
 
 func (h *qemuHandle) Kill() error {
 	// First, try sending a graceful shutdown command via the qemu monitor
-	err := sendQemuShutdown(h.logger, h.monitorPath, h.userPid)
-
-	// If we did not send a graceful shutdown via the monitor socket, we'll
-	// issue an interrupt to the qemu process as a last resort
-	if err != nil {
+	if err := sendQemuShutdown(h.logger, h.monitorPath, h.userPid); err != nil {
```

Comment:
This will log several debug messages relating to graceful shutdown even if the user didn't set the option. We should only attempt the graceful shutdown if the user asked for it.

Reply:
Done!